NossoMercadoLivre/Models/DecodedPassword.cs

```diff
@@ -7,9 +7,9 @@ namespace NossoMercadoLivre.Models
     public class DecodedPassword
     {
         [Required, MinLength(6)]
-        public string Password { get; }
+        public string? Password { get; }
 
-        public DecodedPassword([Required, MinLength(6)] string password)
+        public DecodedPassword([Required, MinLength(6)] string? password)
```

Comment:
Uma curiosidade... quando você manda um new DecodedPassword(...) essas validações Required e MinLenght são aplicadas automaticamente no parâmetro? Se você falar que sim eu vou ficar com muita inveja.

Reply:
Que nada, é só pra deixar pistas, como você falou, eu não gostei muito da abordagem, porq eu acho muito que fica poluído, to usando pra tentar deixar essas pistas de código, como você disse, hehe.